code/modules/library/skill_learning/skillchip.dm

```diff
@@ -602,3 +602,174 @@
+				if(droppage_candidates)
+					var/obj/thing_to_drop = pick(droppage_candidates)
+					thing_to_drop.forceMove(bozo.drop_location())
+			*/ //WHY DOESNTY IT OWRK
+
+			// does not always kill you directly. instead it causes cranial fissure + something to drop from your head. could be eyes, tongue, ears, brain, even implants
+			new /obj/effect/gibspawner/generic/smol(get_turf(bozo), bozo)
+
+			sparks.set_up(15, cardinals_only = FALSE, location = get_turf(src))
+			sparks.start()
+
+			qdel(src)
+		// last chance to stop
+		if(7 to 9)
+			bozo.visible_message(
+				span_danger("[bozo] seems to short circuit!"),
+				span_userdanger("Your brain short circuits!"),
+			)
+			// if they're susceptible to electrocution, confuse them
+			if(bozo.electrocute_act(15, bozo, 1, SHOCK_NOGLOVES|SHOCK_NOSTUN))
+				bozo.adjust_confusion(15 SECONDS)
+				bozo.set_eye_blur_if_lower(10 SECONDS)
+			// but the rest of the effects will happen either way
+			bozo.adjustOrganLoss(ORGAN_SLOT_BRAIN, 20 - get_integrity())
+
+			sparks.set_up(5, cardinals_only = FALSE, location = get_turf(src))
+			sparks.start()
+
+		// brain Smoking. you should probably stop now
+		if(13 to 15)
+			// if already hot, light 'em up
+			var/particle_path = /particles/smoke/steam/mild
+			if(bozo.has_status_effect(/datum/status_effect/temperature_over_time/chip_overheat))
+				bozo.adjust_fire_stacks(11 - get_integrity())
+				bozo.ignite_mob()
+				bozo.visible_message(
+					span_danger("[bozo]'s head lights up!"),
+					span_userdanger("Your head hurts so much, it feels like it's on fire!"),
+				)
+				ASYNC
+					bozo.emote("scream")
+				if(particle_effect?.type == particle_path)
+					return
+				particle_path = /particles/smoke/steam/bad
+			else
+				bozo.visible_message(
+					span_danger("[bozo]'s head starts smoking!"),
+					span_userdanger("Your brain feels like it's on fire!"),
+				)
+
+				// increase smokiness if already smoking
+				if(particle_effect?.type == /particles/smoke/steam/mild)
+					particle_path = /particles/smoke/steam
+				else
+					particle_path = /particles/smoke/steam/mild
+
+			bozo.adjust_confusion(4 SECONDS)
+			bozo.set_eye_blur_if_lower(3 SECONDS)
+
+			particle_effect = new(bozo, particle_path)
+			// roughly head position.
+			// dont know how to make this not hardcoded
+			particle_effect.set_particle_position(-2, 12, 0)
+			bozo.apply_status_effect(/datum/status_effect/temperature_over_time/chip_overheat, 15 SECONDS)
+			QDEL_IN(particle_effect, 15 SECONDS)
+
+			sparks.set_up(10, cardinals_only = FALSE, location = get_turf(src))
+			sparks.start()
+
+		// hey, something isn't right...
+		if(16 to 50)
+			bozo.visible_message(
+				span_warning("[bozo]'s head sparks."),
+			)
+
+			sparks.set_up(rand(1,2), cardinals_only = TRUE, location = get_turf(src))
+			sparks.start()
+
+	return COMPONENT_EMOTE_COOLDOWN_BYPASS
+
+/obj/item/skillchip/acrobatics/proc/charge()
+	allowed_usage++
+
+/obj/item/skillchip/acrobatics/kiss
```

Comment:
This isn't used anywhere. I think you can easily remove this mistake.

Reply:
is there harm in keeping it as an example for mappers and coders. its funny

Reply:
_shrugs_